src/main/kotlin/sipura/alg/GraphProperty.kt

```diff
@@ -219,4 +219,58 @@ object GraphProperty {
         }
         return Pair(degeneracy, ordering)
     }
+
+    /**
+     * Calculates the splittance of the given graph [g].
+     *
+     * The splittance of [g] is the smallest number of edge deletions or edge insertions that are required to
+     * transform [g] into a split graph. That means the splittance of [g] is zero if and only if [g] is a split graph.
+     *
+     * Runtime: O(n + m)
+     * @return the splittance of [g].
+     */
```

Comment:
reference https://en.wikipedia.org/wiki/Splittance#Calculation_from_degree_sequence?
I sadly didn't find an arxiv-link or something similar, just this springer link with a paywall: https://link.springer.com/article/10.1007/BF02579333.

Reply:
I added the link.